rebasehelper/build_tools/koji_tool.py

```diff
@@ -90,8 +90,13 @@ def _scratch_build(cls, source, **kwargs):
         task_dict = cls.koji_helper.watch_koji_tasks(session, [task_id])
         task_list = []
         package_failed = False
+        build_error = ''
         for key in six.iterkeys(task_dict):
             if task_dict[key] == koji.TASK_STATES['FAILED']:
+                try:
+                    info = session.getTaskResult(key)
+                except Exception as result:
```

Comment:
Why `result`? Why not `error` or simply `e`?

Reply:
Also you should catch specific exception type, not general `Exception`.

Reply:
You are correct, I will fix it, as well as the comment, as the exception raised is BuildError, not the one stated.